domains/certificates/Query.API/API/MeasurementsSyncer/SlidingWindowService.cs

```diff
@@ -97,26 +102,33 @@ private static bool IsIncludedInMissingInterval(MeteringPointTimeSeriesSlidingWi
         return window.MissingMeasurements.Intervals.Any(missingInterval => missingInterval.Contains(interval));
     }
 
-    public void UpdateSlidingWindow(MeteringPointTimeSeriesSlidingWindow window, List<Measurement> measurements,
-        UnixTimestamp newSynchronizationPoint)
+    public void UpdateSlidingWindow(MeteringPointTimeSeriesSlidingWindow window, List<Measurement> measurements, UnixTimestamp newSynchronizationPoint)
     {
+        var minimumAgeThreshold = CalculateMinimumAgeThreshold();
```

Comment:
CalculateMinimumAgeThreshold(); <-- Awesome. Exactly this I meant in earlier comment :D

Reply:
Can we resolve this?